Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonFontDialog.cs

```diff
@@ -31,16 +31,6 @@ public string Title
         /// that are currently installed on the system.
         /// </summary>
         public KryptonFontDialog() => _commonDialogHandler = new CommonDialogHandler(true);
-
```

Comment:
Why is this removed

Reply:
It had a duplicate entry, so it wouldn't build